build_tools/meson-openmp-linter.py

```diff
@@ -0,0 +1,106 @@
+# %%
+import json
+import re
+import subprocess
+from pathlib import Path
+
+
+def uses_openmp(targets):
```

Comment:
a bit of comment as what each function does would make reading the code easier, but not a big deal.

Reply:
That was part of the clean-up I had in mind indeed :wink: